src/pocketmine/nbt/NBT.php

```diff
@@ -107,7 +107,13 @@ public static function getItemHelper(CompoundTag $tag){
 			return Item::get(0);
 		}
 
-		$item = Item::get($tag->id->getValue(), !isset($tag->Damage) ? 0 : $tag->Damage->getValue(), $tag->Count->getValue());
+		if($tag->id instanceof IntTag){
+			$item = Item::get($tag->id->getValue(), !isset($tag->Damage) ? 0 : $tag->Damage->getValue(), $tag->Count->getValue());
+		}else{ //PC item save format
```

Comment:
Is it really necessary? Under normal circumstances it's never going to be anything other than an int or string anyway.